data/common.yaml

```diff
@@ -9,3 +9,5 @@ processmaker::dbuser: 'processmaker'
 processmaker::packageensure: 'latest'
 processmaker::casurl: 'localhost'
 processmaker::casenabled: 0
+processmaker::pswsuser: 'pm_ws_user'
```

Comment:
we are likely to have other config file options later. Why not just add an extra_options hash that loops and dumps in the config file so we don't need to edit this module again.

Reply:
While I agree, I'm not taking ownership of this module. I'm doing the minimum amount necessary to get this working for the project to go to production.